debian/clickhouse.limits

```diff
@@ -1,2 +0,0 @@
-clickhouse	soft	nofile	262144
```

Comment:
Why removed?

Reply:
Is it also embedded into binary?

Reply:
It's created by `Install.cpp`
Actually everything is created by `Install.cpp`, everything else is a cruft.